packages/demand-bloc-payment/src/providers/AdyenProvider.ts

```diff
@@ -142,6 +143,11 @@ export class AdyenProvider implements Provider {
 
     handleRefusalResponse(makePaymentResponse.body.payload)
 
+    if (makePaymentResponse.body.payload.resultCode === AUTHORISED) {
+      this.nonce = makePaymentResponse.body.trip_id
+      return ['meta.trip_id', makePaymentResponse.body.trip_id, makePaymentResponse.body.payload.resultCode]
```

Comment:
Why we need to return result code ?

Reply:
To distinct this case in PaymentBloc as in verifyCardWithThreeDSecure  method we use condition for returning resultNonce and in this case this condition doesn't work